storage/postgres/migrations/20220617_initialize_schema.sql

```diff
@@ -1,3 +1,4 @@
+-- +goose Up
```

Comment:
Are these annotations required by goose? Can we get away without them?

Reply:
It seems they are required. https://github.com/pressly/goose#sql-migrations I don't particularly like them either and wish they had it split into two files.